nagioscore.sh

```diff
@@ -22,6 +22,11 @@ function nagiosCore()
 		groupadd -g 9000 nagios
 		groupadd -g 9001 nagcmd
 		useradd -u 9000 -g nagios -G nagcmd -d /usr/local/nagios -c 'Nagios Admin' nagios
+		adduser www-data nagios
+		chown -R nagios:nagios /usr/local/nagios
+		mkdir -p /usr_local/nagios/var/rw
```

Comment:
Is it a typo, or some special thing, that you used /usr_local/ instead of /usr/local/ ?